bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -1,166 +1,438 @@
-        <p><a href="https://blog.mozilla.com/futurereleases/">{{ _('Future of Firefox Blog') }}</a></p>
-      </section>
-    </aside>
+          <span class="note_wrapper">
+            <ul class="section-items tagged">
+              <li {% if not note.tag %}class="untagged"{% endif %} {% if loop.index % 2 != 0 %}class = "gray_note" {% endif %} {% if loop.index % 2 == 0 %}class = "blue_note" {% endif %} id="note-{{ note.id }}">
+                
+                <div class="note_wrap">
+                  {{ note.note|markdown|safe }} 
+                </div>
+              </li>
+            </ul>
+            {% if loop.last %}
+            <br>
+            <br>
+            <br>
+            {% endif %}
+          </span>
+          {% endfor %}
+        </div>
+        <div class="section-wrapper">
+          {% for note in new_features if note.tag =="Resolved" %}
+          {% if loop.first %}
+          <span class="label_wrapper">
+            <p class="note_label">
+              &nbsp;
+              <i class="fa fa-thumbs-up">
+              </i>
+              {{ _('resolved') }} 
+            </p>
+          </span>
+          {% endif %}
+          <span class="note_wrapper">
+            <ul class="section-items tagged">
+              <li {% if not note.tag %}class="untagged"{% endif %} {% if loop.index % 2 != 0 %}class = "gray_note" {% endif %} {% if loop.index % 2 == 0 %}class = "blue_note" {% endif %} id="note-{{ note.id }}">
+                
+                <div class="note_wrap">
+                  {{ note.note|markdown|safe }} 
+                </div>
+              </li>
+            </ul>
+            {% if loop.last %}
+            <br>
+            <br>
+            <br>
+            {% endif %}
+          </span>
+          {% endfor %}
+        </div>
+        {% endif %}
+        <div class="section-wrapper">
+          {% if known_issues %}
+          <section class="notes-section" id="known-issues">
+            {% for note in known_issues %}
+            {% if loop.first %}
+            <div class="label_wrapper">
+              <p class="note_label">
+                &nbsp;
+                <i class="fa fa-ban">
+                </i>
+                {{ _('unresolved') }} 
+              </p>
+            </div>
+            {% endif %}
+            <div class="note_wrapper">
+              <ul class="section-items tagged">
+                <li {% if loop.index % 2 != 0 %}class = "gray_note" {% endif %} {% if loop.index % 2 == 0 %}class = "blue_note" {% endif %}id="note-{{ note.id }}">
+                  
+                  <div class="note_wrap">
+                    {{ note.note|markdown|safe }} 
+                  </div>
+                </li>
+              </ul>
+            </div>
+            {% endfor %}
+            {% endif %}
+          </section>
+        </section>
```

Comment:
These two `</section>` tags don't line up quite right. I think the first one should be placed between the `{% endfor %}` and `{% endif %}` tags. Not sure about the second one. Is it needed?